### PR TITLE
feat: get plausible stats breakdown

### PR DESCRIPTION
**Pull request by levi-zustiak**
# Description

### Issue
Need to fetch the plausible analytic stats for a journey's steps and pass the data to the editor context
_Provide a brief summary of why this task is needed_

[Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/37663055/todos/7446780139)

### Solution
- Created `usePlausibleStatsBreakdownQuery` hook
- Fetch stats break down and pass into editor context

# External Changes
n/a

# Additional information
n/a

### CODE REVIEW

File: libs/journeys/ui/src/libs/EditorProvider/EditorProvider.tsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ import {
 import type { TreeBlock } from '../block'
 import { BlockFields_StepBlock as StepBlock } from '../block/__generated__/BlockFields'
 import { searchBlocks } from '../searchBlocks'
+import { GetPlausibleStatsBreakdown_journeysPlausibleStatsBreakdown as StatsBreakdown } from '../../../../../../apps/journeys-admin/__generated__/GetPlausibleStatsBreakdown'
```
Comment (levi-zustiak) [inline libs/journeys/ui/src/libs/EditorProvider/EditorProvider.tsx:16]: I couldn't get the types to generate for journeys-ui. Is there a proper way to access this type?

Comment (csiyang) [reply libs/journeys/ui/src/libs/EditorProvider/EditorProvider.tsx:16]: Yeah, you can check how we generate journeys field. (Search for `${JOURNEY_FIELDS}`.

But basically

1. Create a `PlausibleStatsBreakDownFields.ts` file in journeys-ui
2. Run codegen for journeys-ui, which will generate the type for you to use in EditorProvider
3. In your hook you can import `${PLAUSIBLE_STATS_BREAKDOWN_FIELDS}` from journeys-ui to use when you create your query

File: libs/journeys/ui/src/libs/EditorProvider/EditorProvider.tsx [hunk 2]
```diff
@@ -85,6 +86,7 @@ export interface EditorState {
    */
   selectedStep?: TreeBlock<StepBlock>
   steps?: Array<TreeBlock<StepBlock>>
+  journeyStatsBreakdown: StatsBreakdownFields[]
```
Comment (csiyang) [inline libs/journeys/ui/src/libs/EditorProvider/EditorProvider.tsx:89]: Lets make this nullable so we don't have to update other components/tests 
```suggestion
  journeyStatsBreakdown?: StatsBreakdownFields[]
```

**Review by csiyang — CHANGES_REQUESTED** [review]

**Comment by csiyang** [comment]
closing in favour of https://github.com/JesusFilm/core/pull/2841

**Pull request closed without merging**